src/hooks/useScroll.ts

```diff
@@ -0,0 +1,14 @@
+import { RefObject, useEffect } from 'react';
+
+const useScroll = (ref: RefObject<HTMLElement>, scrollCallback: () => void) => {
+  useEffect(() => {
+    if (ref.current === null) {
+      return;
+    }
+
+    ref.current.addEventListener('scroll', scrollCallback);
```

Comment:
```suggestion
    ref.current.addEventListener('scroll', scrollCallback, { passive: true });
```
패시브 옵션 사용해서 이벤트 감시 리소스를 줄여주면 좋을 것 같아요!

Reply:
오! 좋습니닷👍

Reply:
f482f12 에서 반영했습니다!